addons/website_forum/views/website_forum.xml

```diff
@@ -1426,7 +1426,7 @@
 
     <div t-attf-class="o_wforum_author_box d-inline-flex #{display_info and 'o_show_info'} #{compact and 'o_compact align-items-center'} #{bio_popover_data and 'o_wforum_bio_popover'}"
          t-att-data-content="bio_popover_data">
-        <t t-set="user_profile_url" value="#"/>
+        <t t-set="user_profile_url" t-value="'#'"/>
```

Comment:
```suggestion
        <t t-set="user_profile_url" t-valuef="#"/>
```

Reply:
@rdeodoo Is this really necessary? Because below is using t-value.

Reply:
That's not the same case, it is mixing string and python.
But when your value is plain text only, using `valuef` is better for readability, it's its sole purpose, not having to wrap the string in single quote for no reason.

But it's not wrong or right, it's doing the same thing (`t-value="'ok'"` / `t-valuef="ok"`), one is just clearer. So when introducing new line or modifying some, always good to keep it in mind :+1: 

Same for any `t-attf-xx` instead of `t-att-xx`

Reply:
Ok thanks :+1: